docs/docsite/rst/community/steering/steering_committee_membership.rst

```diff
@@ -65,7 +65,7 @@ Voluntarily leaving the Steering Committee
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 A Committee member can voluntarily leave the Committee.
-In this case, they notify the other members, create an issue in the `Community Topics <https://github.com/ansible-community/community-topics/issues>`_ repository announcing the resignation, and after that, they are no longer considered Committee members.
+In this case, they notify the other members, create an issue in the `Community Topics <https://forum.ansible.com/tags/c/project/7/community-wg>`_ repository announcing the resignation, and after that, they are no longer considered Committee members.
```

Comment:
```suggestion
In this case, the member notifies the other members, creates a `Community Topic <https://forum.ansible.com/tags/c/project/7/community-wg>`_ announcing their resignation, and after that, they are no longer considered a Committee member.
```

Reply:
i deliberately didn't touch it here to change it in a separate PR (and also there's another PR about updating the links, so to avoid conflict). OK, let's merge it here, then i fix the conflicts if appear.